test-fixtures/erroring-test/src/code.sh

```diff
@@ -0,0 +1 @@
+#!/bin/bash
```

Comment:
I agree with @godp1301 , this file should not be part of this review.  Is this empty script used somewhere?